### PR TITLE
Update `_anisotropic` hint suffix in Shading language

### PR DESCRIPTION
**Pull request by WildRackoon**
Corrected '_aniso' to '_anisotropic' in shading_language.rst.
I am Unsure if this was intentional to respect some table width maximum.

### CODE REVIEW

File: tutorials/shaders/shader_reference/shading_language.rst [hunk 1]
```diff
@@ -755,33 +755,33 @@ engine renders in linear color space.
...
+| **sampler2D**        | hint_anisotropy                                 | As flowmap, default to right.                                               |
++----------------------+-------------------------------------------------+-----------------------------------------------------------------------------+
+| **sampler2D**        | hint_roughness[_r, _g, _b, _a, _normal, _gray]  | Used for roughness limiter on import (attempts reducing specular aliasing). |
+|                      |                                                 | ``_normal`` is a normal map that guides the roughness limiter,              |
+|                      |                                                 | with roughness increasing in areas that have high-frequency detail.         |
++----------------------+-------------------------------------------------+-----------------------------------------------------------------------------+
+| **sampler2D**        | filter[_nearest, _linear][_mipmap][_anisotropic]| Enabled specified texture filtering.                                        |
```
Comment (Calinou) [inline tutorials/shaders/shader_reference/shading_language.rst:781]: The table width needs to be extended by 1 character, as the last character should *not* touch the pipe character (`|`).

Comment (WildRackoon) [reply tutorials/shaders/shader_reference/shading_language.rst]: You are right, fixed

**Review by clayjohn — APPROVED** [review]
Looks like the option was renamed in https://github.com/godotengine/godot/pull/55951 and https://github.com/godotengine/godot/pull/55969 and we just forgot to update the docs

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Pull request merged**